src/findlib.ml

```diff
@@ -45,22 +45,24 @@ module Rules = struct
     ; add_rules : Rule.t list
     }
 
-  let interpret t ~preds =
+  let interpret' t ~preds =
```

Comment:
I find the quote suffix not very informative. I don't have a better name in mind thought, what about exposing only one version of this function, the one returning an option and using `Option.value ... ~default:""` at call site wherever needed?